docs/pages/develop/user-interface/fonts.mdx

```diff
@@ -113,16 +146,18 @@ In the example below, the `<Text>` uses the `Inter-Black` as the font family nam
 
 #### How to determine which font family name to use
 
-On Android, the font family name is the same as of the font file (without the extension). On iOS, the font family name is read from the font file itself.
+- If you provide fonts as an array of file paths: As described above, on Android, the file name (without the extension) becomes the font family name. On iOS, the font family name is read from the font file itself. We recommend naming the font file same as its [PostScript name](#what-is-postscript-name-of-a-font) so the font family name is consistent on both platforms.
 
-We recommend naming the font file same as its [PostScript name](#what-is-postscript-name-of-a-font) so the font family name is consistent on both platforms.
+- If you use the object syntax: provide the "Family Name". This can be found in the Font Book app on macOS, [fontdrop.info](https://fontdrop.info/) or other programs.
```

Comment:
```suggestion
- If you use the object syntax, provide the "Family Name". This can be found in the Font Book app on macOS, [fontdrop.info](https://fontdrop.info/) or other programs.
```